src/Form/Field/EntriesField.php

```diff
@@ -63,14 +69,9 @@ public function fill(mixed $value): static
 		return $this;
 	}
 
-	public function form(array $values = []): Form
+	public function form(): Form
 	{
-		$form = new Form(
-			fields: [$this->field()],
-			model:  $this->model
-		);
-
-		return $form->fill(input: $values);
+		return $this->form;
```

Comment:
```suggestion
		return $this->form ??= new Form(
			fields: [$this->field()],
			model: $this->model
		);
```

I would create this here lazily instead of in the constructor.